src/pudl/metadata/classes.py

```diff
@@ -1914,22 +1914,26 @@ class DatasetteMetadata(PudlMeta):
     def from_data_source_ids(
         cls,
         output_path: Path,
-        data_source_ids: Iterable[str] = [
+        data_source_ids: list[str] = [
             "pudl",
-            "ferc1",
             "eia860",
-            "eia861",
             "eia860m",
+            "eia861",
             "eia923",
+            "ferc1",
```

Comment:
non-blocking: What would it take to change these data_source_ids to `fercX_dbf` also?

Reply:
This would be a significant change. Right now the DBF and XBRL data are both subsumed within the higher level conceptual data source known and `ferc1` (or whatever) and each conceptual data source has a bunch of metadata associated with it. Breaking down those objects into smaller pieces that understand the internal complexities of where the data for different years are coming from would be substantial work, since the current idea of a "data source" shows up in a lot of different places.

That said, looking at all the complexity that's involved in bringing together the information required to document the databases in datasette, there's clearly room for improvement in our overall metadata modeling.